.github/workflows/hard_reset.sh

```diff
@@ -1,23 +0,0 @@
-#!/bin/sh
```

Comment:
Why are we deleting these?

Reply:
I moved it. I think it's better to keep all scripts in one place.